### PR TITLE
[miio] support sending openclose commands

### PR DESCRIPTION
**Pull request by marcelrv**
Note. Depending on the device, openclose to be send as number or on/off
Signed-off-by: Marcel Verpaalen <marcel@verpaalen.com>

### CODE REVIEW

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java [hunk 1]
```diff
@@ -175,6 +175,26 @@ public void handleCommand(ChannelUID channelUID, Command receivedCommand) {
                             command = new DecimalType(((QuantityType<?>) command).toBigDecimal());
                         }
                     }
+                    if (paramType == CommandParameterType.OPENCLOSENUMBER) {
+                        if (command instanceof OpenClosedType) {
+                            value = new JsonPrimitive(command == OnOffType.ON ? 1 : 0);
```
Comment (lolodomo) [inline bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:180]: Looks weird. If command is instance of OpenClosedType, its value will never be OnOffType.ON.

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java [hunk 2]
```diff
@@ -175,6 +175,26 @@ public void handleCommand(ChannelUID channelUID, Command receivedCommand) {
...
+                        } else {
+                            value = new JsonPrimitive(
+                                    ("ON".contentEquals(command.toString()) || "1".contentEquals(command.toString()))
+                                            ? 1
+                                            : 0);
+                        }
+                    }
+                    if (paramType == CommandParameterType.OPENCLOSE) {
+                        if (command instanceof OpenClosedType) {
+                            value = new JsonPrimitive(command == OnOffType.ON ? "on" : "off");
```
Comment (lolodomo) [inline bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java:190]: Same as above

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java (hunk 1 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java]: totally right! too much copy pasting:-(

File: bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java (hunk 2 above)
Comment (marcelrv) [reply bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/handler/MiIoBasicHandler.java]: also fixed

**Review by lolodomo — APPROVED** [review]
LGTM

**Pull request merged**